hpOneView/resources/servers/id_pools_ranges.py

```diff
@@ -0,0 +1,209 @@
+                in OneView, just stops waiting for its completion.
+
+        Returns:
+            dict: Created range
+        """
+        return self._client.create(resource, timeout=timeout)
+
+    def get(self, id_or_uri):
+        """
+        Gets range.
+
+        Using the allocator and collector associated with the range, IDs may be allocated from or collected back to the
+        range.
+
+        Args:
+            id_or_uri: Could be either the range id or uri
+
+        Returns:
+            dict: range
+        """
+        return self._client.get(id_or_uri)
+
+    def enable(self, information, id_or_uri, timeout=-1):
+        """
+        Enables or disables a range.
+
+        Args:
+            information (dict): information to update.
+            id_or_uri: id or uri of range
+            timeout:
+                Timeout in seconds. Wait task completion by default. The timeout does not abort the operation
+                in OneView, just stops waiting for its completion.
+
+        Returns:
+            dict: Updated resource
+
+        """
+        if id_or_uri:
+            uri = self._client.build_uri(id_or_uri)
+
+        data = {
+            "type": "Range",
+            "enabled": information["enabled"]
+        }
+
+        return self._client.update(data, uri, timeout=timeout)
+
+    def delete(self, resource, force=False, timeout=-1):
+        """
+        Deletes range.
+
+        Args:
+            resource (dict): object to delete
+            force (bool):
+                 If set to true the operation completes despite any problems with
+                 network connectivity or errors on the resource itself. The default is false.
+            timeout:
+                Timeout in seconds. Wait task completion by default. The timeout does not abort the operation
+                in OneView, just stops waiting for its completion.
+
+        Returns:
+            dict: Details of associated resource
+        """
+        return self._client.delete(resource, force=force, timeout=timeout)
+
+    def get_allocated_fragments(self, resource, count=-1, start=0):
+        """
+        Gets all fragments that have been allocated in range.
+
+        Args:
+            resource: resource
+            count:
+                 The number of resources to return. A count of -1 requests all the items. The actual number of items in
+                 the response may differ from the requested count if the sum of start and count exceed the total number
+                 of items, or if returning the requested number of items would take too long.
+            start:
+                The first item to return, using 0-based indexing. If not specified, the default is 0 - start with the
+                first available item.
+
+        Returns: the list of IDs
+
+        """
+        uri = resource[
+            'uri'] + "/allocated-fragments?start={0}&count={1}".format(start, count)
```

Comment:
We think the methods that are related to a specific resource should always receive a resource identifier (`id_or_uri`). So you could use: `self._client.build_uri(id_or_uri) + "/allocated-fragments?start={0}&count={1}".format(start, count)`